src/form/Form.js

```diff
@@ -196,6 +215,49 @@
         this.btntd
             .attr("colspan", this._maxCols * 2)
         ;
+        
+        // Disable Submit unless there is data
+        if (!this.allowEmptyRequest()) {
+            this.inputs().forEach(function(input, idx) {
```

Comment:
becuase what if the form is updated dynamically and new event handlers need to be added (i always think along those lines)